### PR TITLE
Update Deploy a Worker Service to Azure article

### LINKED ISSUE #25865: Address issues from user study for "Deploy Worker Service to Azure"
There are several primary issues to address from watching various user studies.

Article: [Deploy a Worker Service to Azure](https://docs.microsoft.com/en-us/dotnet/core/extensions/cloud-service)

#### Developer experience

_When the user has selected **Visual Studio**:_

- [x] Consider **Enable Docker** checkbox in create project wizard, rather than right-click project node
- [x] Try to make it more clear to the user that they need to right-click the project node, as they kept clicking the solution

_When the user has selected **Visual Studio Code**:_

- [x] Move the Azure admin account instruction down into the VS Code area rather than have it at the top like that. Extend the sentence with the Docker extension link and add "and the azure account extension" and link to it, too.

_When the user has selected **Command line interface**:_

- [x] The _Dockerfile_ is **_not_** created for you, make that more obvious
- [x] The _Dockerfile_ does _**not**_ have a file extensions
- [x] The CLI path does **_not_** have Azure CLI commands for deploying ACR and ACI bits -- add these

_General updates:_

The Azure Portal instructions should always read LTR, especially when the UI from the portal reads that way.

- [x] Instead of, "Select **Azure Container Registry** as the **Image source**" say "For the **Image source**, select "**Azure Container Registry**".
- [x] Overall it seems as though there are sections that make assumptions about transition phases, where the user might have to wait for something to happen before continuing - the use of imagery could help here. Specifically, waiting for the Azure Portal to deploy a resource.

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Address all the various points of feedback from the "Deploy Worker Service to Azure" article.

Fixes #25865

✔️ [Preview: Deploy a Worker Service to Azure](https://review.docs.microsoft.com/en-us/dotnet/core/extensions/cloud-service?branch=pr-en-us-26080)

### CODE REVIEW

File: docs/core/extensions/cloud-service.md [hunk 1]
```diff
@@ -53,14 +54,29 @@ If you didn't select this option, no worries &mdash; you can still add it now. I
...
 
-In Visual Studio Code, you'll need the [Docker extension](https://code.visualstudio.com/docs/containers/overview) installed. Open the Command Palette, and select the **Docker: Add Docker files to workspace** option. If prompted to **Select Application Platform** choose **.NET: Core Console**. If prompted to **Select Project**, choose the Worker Service project you created. When prompted to **Select Operating System**, choose the first listed OS. When prompted whether or not to **Include optional Docker Compose files**, select **No**.
+In Visual Studio Code, you'll need the [Docker extension](https://code.visualstudio.com/docs/containers/overview) and the [Azure Account extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) installed. Open the Command Palette, and select the **Docker: Add Docker files to workspace** option. If prompted to **Select Application Platform** choose **.NET: Core Console**. If prompted to **Select Project**, choose the Worker Service project you created. When prompted to **Select Operating System**, choose the first listed OS. When prompted whether or not to **Include optional Docker Compose files**, select **No**.
```
Comment (bradygaster) [inline docs/core/extensions/cloud-service.md:57]: 💯

File: docs/core/extensions/cloud-service.md [hunk 2]
```diff
@@ -53,14 +54,29 @@ If you didn't select this option, no worries &mdash; you can still add it now. I
...
 
 :::zone-end
 
-Docker support requires a *Dockerfile*. This file is a set of comprehensive instructions, for building your .NET Worker Service as a Docker image. The following is an example *Dockerfile*, and should exist at the root directory of the project file:
+Docker support requires a *Dockerfile*. This file is a set of comprehensive instructions, for building your .NET Worker Service as a Docker image. The *Dockerfile* is a file *without* a file extension. The following is an example *Dockerfile*, and should exist at the root directory of the project file.
+
+:::zone target="docs" pivot="cli"
+
+With the CLI, the *Dockerfile* is **not** created for you. You'll need to copy its contents into a new file name *Dockerfile*, and again this file should be in the root directory of the project.
```
Comment (bradygaster) [inline docs/core/extensions/cloud-service.md:65]: 👍

File: docs/core/extensions/cloud-service.md [hunk 3]
```diff
@@ -212,32 +262,76 @@ Expand the **REGISTRIES** node, select **Azure**, your  subscription > the conta
...
-1. Select **Azure Container Registry** as the **Image source**.
+1. For **Image source**, select **Azure Container Registry**.
 1. Select the **Registry** by the name provided in the previous step.
 1. Select the **Image** and **Image tag**.
 1. Select **Review + create**.
 1. Assuming **Validation passed**, select **Create**.
 
+It may take a moment for the resources to be created, once created select the **Go to resource** button.
+
 For more information, see [Quickstart: Create an Azure container instance](/azure/container-instances/container-instances-quickstart-portal).
```
Comment (bradygaster) [inline docs/core/extensions/cloud-service.md:300]: If possible, i think it'd be good for you to provide folks with the CLI command required to enlist their Azure Sub to be able to use ACI. i don't know if ACI is one of the automatically-enabled providers; in a lot of scenarios where I'm testing this scenario in new Azure subs, I've had to enable the ACI RP. If that's lacking from the article folks may get lost once they try to run their container image as an instance. Small fries, probably, and I'm sure the ACI docs you link off to would suffice.

**Review by bradygaster — APPROVED** [review]

File: docs/core/extensions/cloud-service.md (hunk 3 above)
Comment (IEvangelist) [reply docs/core/extensions/cloud-service.md:300]: I added all the AZ CLI commands, and tested them. This should be golden now.

**Pull request merged**